.circleci/config.yml

```diff
@@ -1,4 +1,10 @@
-version: 2
+version: 2.1
+
+parameters:
+  check_deployability:
+    type: boolean
+    default: false
```

Comment:
`check_deployability` is only set to `true` from vets-api invoking the `can-i-deploy` workflow.

When `true`, it runs only the `can-i-deploy` workflow and not any others. Conversely, by default, this excludes `can-i-deploy` workflow from the regular build pipeline because it's an async job.